### PR TITLE
Adds query to support new triggered A&A display.

### PR DESCRIPTION
**Pull request by oceanzus**
Adds REDIS_URL config var to init.

### CODE REVIEW

File: ooiservices/app/__init__.py [hunk 1]
```diff
@@ -38,6 +38,10 @@ def create_app(config_name):
         env.from_yaml(os.path.join(basedir, 'config.yml'))
     celery.conf.update(BROKER_URL=app.config['REDIS_URL'],
                 CELERY_RESULT_BACKEND=app.config['REDIS_URL'])
+    if app.config['REDIS_CACHE']:
+        cache.config = {'CACHE_TYPE': 'redis', 'CACHE_REDIS_URL': app.config['REDIS_URL']}
+    else:
+        cache.config = {'CACHE_TYPE': 'simple'}
```
Comment (FBRTMaka) [inline ooiservices/app/__init__.py]: @oceanzus this isn't the correct way, we aren't giving them the option to choose what cache type they are using.

**Comment by FBRTMaka** [comment]
Hey @oceanzus, you'll need to roll back the changes on this for __init__.py and config.yml; they aren't the correct way to use redis for out application.

**Comment by oceanzus** [comment]
@FBRTMaka Can you be more specific as to what is not correct?

**Comment by FBRTMaka** [comment]
sure:

From __init__.py
+cache = Cache()


 if app.config['REDIS_CACHE']:
+        cache.config = {'CACHE_TYPE': 'redis', 'CACHE_REDIS_URL': app.config['REDIS_URL']}
+    else:
+        cache.config = {'CACHE_TYPE': 'simple'}

and from config.yml:

+    REDIS_CACHE: True

**Comment by FBRTMaka** [comment]
@oceanzus basically, just revert the init.py and config.yml

**Comment by FBRTMaka** [comment]
@oceanzus , @DanielJMaher that's actually the better approach - I do like the URL configuration for Redis.

**Pull request merged**